test/main.js

```diff
@@ -634,9 +634,9 @@ describe('node-fetch', () => {
 			const read = async body => {
 				const chunks = [];
 
-				if (process.version < 'v14') {
-					// In Node.js 12, some errors don't come out in the async iterator; we have to pick
-					// them up from the event-emitter and then throw them after the async iterator
+				if (process.version < 'v14.15.2') {
```

Comment:
Umm, when have it ever been possible to compare strings like this?

Reply:
Well, it will compare lexicographically, so as long as Node.js doesn't release a version `14.100.0` or `100.0.0` I think it should work...

But we could use e.g. [`semver-compare`](https://www.npmjs.com/package/semver-compare) instead ☺️

Reply:
> But we could use e.g. semver-compare instead ☺️

like fewer dependencies. found this on https://stackoverflow.com/a/55466325/1008999